### ISSUE TITLE
Save/Restore edit cursor position doesnt work inside custom action associated with mouse modifiers

### ISSUE THREAD

**Issue by MichaelPilyavskiy** [Jeff0S/sws#719]
Save/Restore edit cursor position doesnt work inside custom action associated with mouse modifiers.
If I understand right, it is because of when I click mouse, first it set edit cursor position, and then store edit cursor position.

Got it from custom action (mouse modifier - track - left click):
1 Save edit cursor position slot 1 // should store current position, but it doesnt
2 Save selected tracks/selected items slot 1
3 SWS Select first item on selected track
4 Copy item
5 Move edit cursor to mouse cursor
6 Paste item
7 Restore selected tracks/selected items slot 1
8 Restore edit cursor position slot n // cursor move to start of copied item

**Comment by Breeder** [comment]
Please upload the problematic macro so I can have a look. Thanks!

**Comment by MichaelPilyavskiy** [comment]
https://drive.google.com/file/d/0B6UUCJtTJFtoaXY1ekNkZVFBVW8/view?usp=sharing

**Comment by Breeder** [comment]
Yes, as you suspected, as soon as you click in the arrange and SWS/BR save action is called, GetCursorPositionEx(NULL) will return time position of where mouse click happened.

Nothing we can do about it from SWS really...
